src/scss/components/file/mixins/hxFileControl.scss

```diff
@@ -0,0 +1,9 @@
+@import "../config";
+
+@mixin hxFileControl(
```

Comment:
Please reformat mixin to make it easier to read:

```scss
@mixin hxFileControl($pseudo-state: null) {
  @if $pseudo-state == focus {
    box-shadow: $focus-glow;
   }
  @else {
    // no additional styles applied 
   }
}
```

Reply:
Changes have been done, please review.